demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -872,6 +874,15 @@ def _collect_classifiers_dependencies(pack_classifiers: list, id_set: dict, verb
                     packs_found_from_incident_types)
                 classifier_dependencies.update(pack_dependencies_data)
 
+            if classifier_data.get('definitionId'):
+                packs_found_from_generic_types = PackDependencies._search_packs_by_items_names(
+                    related_types, id_set['GenericTypes'], exclude_ignored_dependencies)
+
+                if packs_found_from_generic_types:
+                    pack_dependencies_data = PackDependencies._label_as_mandatory(
+                        packs_found_from_generic_types)
+                    classifier_dependencies.update(pack_dependencies_data)
```

Comment:
you should either check for types in the generics or in incident types - not both as they might have the same name theoretically.

Reply:
Done